### ISSUE TITLE
SolrIndexTask fails with DB failure

### ISSUE THREAD

**Issue by phptek** [Firesphere/silverstripe-solr-search#138]
**Describe the bug**

I was using `dev-master#7c78fb8c` which worked fine. I then inadverently upgraded to `dev-master#48317a88e` and from that point on, running the task fails with the trace below

```
vagrant@project: (solr-compat) $> ./vendor/bin/sake dev/tasks/SolrIndexTask
Running Task Solr Index update

ERROR [Warning]: Error while sending STMT_PREPARE packet. PID=9400
IN GET dev/tasks/SolrIndexTask
Line 66 in /vagrant/vendor/silverstripe/framework/src/ORM/Connect/MySQLiConnector.php

...
...

SilverStripe\ORM\Queries\SQLExpression->execute()
SQLSelect.php:648

SilverStripe\ORM\Queries\SQLSelect->count(DISTINCT "SiteTree"."ID")
DataQuery.php:450

SilverStripe\ORM\DataQuery->count()
DataList.php:868

SilverStripe\ORM\DataList->count()
SolrIndexTask.php:243

...
```

**To Reproduce**

* [OK] Start from a clean Solr server (Zero cores and no .solr directory on the F/S)
* [OK] Run dev/tasks/SolrConfigureTask
* [FAIL] dev/tasks/SolrIndexTask

**Expected behavior**

No stacktrace! :-)

**Desktop (please complete the following information):**

SilverStripe 4.4.3 + PHP7.2 + MySQL 5.7.27

**Smartphone (please complete the following information):**

N/A

**Additional context**

N/A

**Comment by Firesphere** [comment]
Hi @phptek 
This should be resolved now. It's due to the implementation of `PCNTL`, could you try again with the latest tagged release?

**Comment by Firesphere** [comment]
Follow-up:
Please stick to tagged releases, master should not be considered 100% stable. Unless you're happy to test out new features, of course.

**Comment by Firesphere** [comment]
p.s.

You can use GitHub flavoured markdown to add a collapsed block.

I think it's something like this:
<details>
  <summary>Click to expand!</summary>
  
  ## Heading
  1. A numbered
  2. list
     * With some
     * Sub bullets
</details>

You can use that, to put large stacktraces in, without cluttering the view for the reader :)

**Comment by Firesphere** [comment]
Should be fixed in `master`, please give it a try?

**Comment by Firesphere** [comment]
Fixed in release 0.8.3
